src/ring_theory/finiteness.lean

```diff
@@ -8,6 +8,7 @@ import algebra.algebra.restrict_scalars
 import algebra.algebra.subalgebra.basic
 import group_theory.finiteness
 import ring_theory.ideal.operations
+import ring_theory.ideal.quotient
```

Comment:
I think you can avoid this `import` by tweaking 1 proof a tiny little bit.

Reply:
Happy for me to do that as a follow-up? I agree that your proof in #18530 is nice here, but figure that it's fine to remain either in your PR or in another spin-off I can make tomorrow.

Reply:
Yes, please create a follow-up PR.